KIPs/kip-160.md

```diff
@@ -0,0 +1,80 @@
+---
+kip: 160
+title: An Update of Treasury Fund Rebalancing
+requires: 103
+---
+
+## Simple Summary
+An update of treasury fund rebalancing.
+
+## Abstract
+The treasury fund rebalancing is updated, resulting in updates to the treasury rebalance contract v2 and the related core logic.
+
+## Motivation
+According to kip103, the treasury rebalancing refers to the act of burning existing fund balances and minting new funds. This event happens at a reserved time, such as a hard fork block number. Through the TreasuryRebalance contract, the disclosure of the treasury fund activities can be conducted transparently and verifiably.
+
+But in kip103, the rebalancing only considers the case when the total balance of funds decreases. This proposal expands to the general cases so that we don't have to consider whether the final result is burn or mint. The other improvement is about making rebalanceBlocknumber in the RebalanceContract editable. The rebalanceBlocknumber should be matched with the related hardfork block number. However, it is not updatable even hard fork block number can be updated.
```

Comment:
Could we summarize the problems somewhere? perhaps at the beginning of this paragraph?

However, there are a few shortcomings of KIP103; (1) it only permits the decrease in the total balance of the funds, (2) its hardfork number is immutable, and (3) it uses confusing variable names such as Retired/Newbie.

Reply:
@ian0371 I've revised the Motivation section as you adviced. Can you take a look? https://github.com/klaytn/kips/pull/160/commits/96c88e40738a2f494e3c3893c5265f87f09f1aa2